_posts/2015-12-16-adding-csrf-protection-to-your-rails-backbone-app.markdown

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "Adding Csrf-Protection to your Rails-Backbone App"
+date: 2015-12-16 03:55:00
+categories: ["rails", "backbone", "security"]
+author: "schmierkov"
+---
+
+When integrating Backbone.js in your Rails App, you might face the problem of the inability to verify the CSRF-Token.
+
+The CSRF Protection secures your app with a token to make sure, that the person who is interacting with it is you. So you should not turn it off, except you know what you are doing. For more informations on this Topic, check out the [Rails Security Guide](http://guides.rubyonrails.org/security.html#cross-site-request-forgery-csrf).
```

Comment:
@schmierkov I'd change `except you know what you are doing` for `unless you know what you are doing`. And `informations` should be just `information`. :+1: